incubator/spark-history-server/templates/deployment.yaml

```diff
@@ -0,0 +1,138 @@
+{{- if .Values.createNamespace }}
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: {{ .Values.namespace }}
+{{- end }}
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: spark-history-server
+  namespace: {{ .Values.namespace }}
+---
+apiVersion: extensions/v1beta1
```

Comment:
Can you use the `Deployment` API version `apps/v1`. All supported versions of Kubernetes by the community support that API version.